widget/embedded/src/store/slices/data.ts

```diff
@@ -197,6 +199,14 @@ export const createDataSlice: StateCreator<
 
     return list;
   },
+  findToken: (token) => {
+    if (!token) {
```

Comment:
`token` is a mandatory value for this method. so please remvoe `undefined` from input and remove this `if` here.